foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/data-lineage-migration.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Data Lineage upgrade migration
+
+  Scenario: Migrate the Data Lineage module files with the new package name when the VERSION_AISSEMBLE is less than the 1.7.0.
```

Comment:
A: This scenario is written oddly to me. There is no variable `VERSION_AISSEMBLE` that we're updating. Scenarios should focus on the functional goals.  I might rewrite these to be something more like:
```
Scenario: Update Data Lineage package name
  Given a Java file references the package "com.boozallen.aiops.data.lineage"
  When the 1.7.0 data lineage package migration executes
  Then the references are updated to "com.boozallen.aissemble.data.lineage"
```